config-ui/src/styles/nav.scss

```
@@ -17,11 +17,43 @@
   position: fixed;
   top: 0;
   border-radius: 0;
-  background: #ffffff;
   box-shadow: none;
   z-index: 1;
 }
 
+.navbarMenuButton{
```

This css rule is missing a space before the opening bracket `{`
```suggestion
.navbarMenuButton {
```